### ISSUE TITLE
Various Issues on Windows 

### ISSUE THREAD

**Issue by hslee2008** [neurosity/neurosity-sdk-python#16]
```bash
Traceback (most recent call last):
  File "C:\Users\user\Desktop\code\bci\new-command\index.py", line 4, in <module>
    neurosity = NeurositySDK({
                ^^^^^^^^^^^^^^
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\neurosity\neurosity.py", line 27, in __init__
    signal.signal(signal.SIGHUP, self.exit_handler)
                  ^^^^^^^^^^^^^
AttributeError: module 'signal' has no attribute 'SIGHUP'
Exception ignored in atexit callback: <bound method NeurositySDK.exit_handler of <neurosity.neurosity.NeurositySDK object at 0x000001908F13A330>>
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\neurosity\neurosity.py", line 30, in exit_handler
    self.remove_client()
  File "C:\Users\user\AppData\Local\Programs\Python\Python312\Lib\site-packages\neurosity\neurosity.py", line 60, in remove_client
    client_id = self.client_id
                ^^^^^^^^^^^^^^
AttributeError: 'NeurositySDK' object has no attribute 'client_id'
```

Even running the example codes, the above error appears.

```python
from neurosity import NeurositySDK
from dotenv import load_dotenv
import os

load_dotenv()

neurosity = NeurositySDK({
    "device_id": os.getenv("NEUROSITY_DEVICE_ID")
})

neurosity.login({
    "email": os.getenv("NEUROSITY_EMAIL"),
    "password": os.getenv("NEUROSITY_PASSWORD")
})
```

**Comment by sean1832** [comment]
Did you solved this issue? I am also getting this and have no idea what to do.

**Comment by hslee2008** [comment]
Nope. Just gave up. Neurosity doesn't seem too interested in trying to solve this problem.

**Comment by andrewjaykeller** [comment]
Please test latest release 2.2.0 on pypi for the fix from #15
